### DISCUSSION TITLE
Bi-lingual Recognition

### DISCUSSION THREAD

**Discussion by bely66** [open-mmlab/mmocr#1461] [Ideas]
I'm thinking of using SAR for English + Non-Latin Language.
I don't think it's impossible, but I'm facing some ambiguities.

would it be hard to know the criteria which was used when training English + Chinese, For ex. what was the split size between them? the whole dataset size? would the accuracy be affected?

**Comment by gaotongxiao** [comment]
We're also a newbie to multilingual recognition. I think this issue could be left as a general discussion.
